exporter/debugexporter/README.md

```diff
@@ -32,6 +32,7 @@ The following settings are optional:
   messages are logged (every Mth message is logged). Refer to [Zap
   docs](https://godoc.org/go.uber.org/zap/zapcore#NewSampler) for more details.
   on how sampling parameters impact number of messages.
+- `use_internal_logger` (default = `true`): if set, uses the collector's internal logger for output. See [below](#using-the-collectors-internal-logger) for description.
```

Comment:
If we want this component to have a separate logger, do we expect people to want to configure this separate logger? If so, should we have a `logger` section instead to group features related to configuring the logger?

Reply:
I actually think that what the exporter uses under the hood to output data should be hidden from the user as an implementation detail. The user should be able to configure the output of the exporter without thinking if a logging library is being used under the hood or something else. This frees us from being tied to a specific implementation of the output.

Reply:
I would envision maybe an `output` section that would configure whether you want the resource or scope to be output as separate lines. Potentially also to configure the output more, like define whether timestamps should be output, log severity, etc.